plugins/flytekit-pandera/flytekitplugins/pandera/pandas_transformer.py

```diff
@@ -100,7 +99,7 @@ def to_literal(
         else:
             html = renderer.to_html(val, schema)
         finally:
-            Deck(renderer._title, html)
+            Deck(renderer._title, html).publish()
```

Comment:
Given this is new flytekit API, I think this requires a bump in the minimum flytekit version?

https://github.com/flyteorg/flytekit/blob/master/plugins/flytekit-pandera/setup.py#L7